apps/src/blockly/addons/cdoVariables.js

```diff
@@ -1,9 +1,197 @@
+import {caseInsensitiveCompare} from '../utils';
+
 export default function initializeVariables(blocklyWrapper) {
-  // TODO - required for sprite variable blocks
-  blocklyWrapper.Variables.registerGetter = function (category, blockName) {};
+  // Functionality pulled from https://github.com/code-dot-org/blockly/blob/main/core/utils/variables.js
+  // Omits the following functions from the original file: 1. renameVariable, 2. deleteVariable,
+  // 3. flyoutCategory, 4. getGetter, 5. getSetter
+  blocklyWrapper.Names.PREFIX_ = '$_';
+
+  blocklyWrapper.Variables.NAME_TYPE = 'VARIABLE';
+  blocklyWrapper.Variables.DEFAULT_CATEGORY = 'Default';
+  blocklyWrapper.Variables.getters = {
+    Default: 'variables_get',
+  };
+  blocklyWrapper.Variables.setters = {
+    Default: 'variables_set',
+  };
+
+  blocklyWrapper.Variables.registerGetter = function (category, blockName) {
```

Comment:
Since the implementation is pretty trivial and the calls to `registerGetter` are still there, is it better to file a follow-up task to remove support for these after the migration?

Reply:
That makes sense to me! I agree that your implementation is totally reasonable. It would be great to audit some of our lab code and eventually remove stuff we've been supporting for no real reason.

Reply:
Added a ticket and linked it in the comment.